### PR TITLE
Add ghq extension

### PR DESCRIPTION
**Pull request by konojunya**
## Description

Raycast Extension for [ghq](https://github.com/x-motemen/ghq) `list` and `get`.

## Screencast

Raycast-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/ghq/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "subtitle": "ghq",
+      "description": "Open the search results in VSCode or another application. For example, Finder or your terminal app",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "label": "ghq root path",
+      "name": "GHQ_ROOT_PATH",
+      "title": "ghq root path",
```
Comment (pernielsentikaer) [inline extensions/ghq/package.json:25]:
```suggestion
      "title": "ghq Root Path",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- I don't have `Visual Studio Code` installed so maybe it would be better to ask the user in preferences for editor with [appPicker](https://developers.raycast.com/api-reference/preferences#return)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/ghq/package.json (hunk 1 above)
Comment (konojunya) [reply extensions/ghq/package.json]: fix: https://github.com/raycast/extensions/pull/10768/commits/e33797621cd3232c1cd10178c38487c92544b32e

**Comment by konojunya** [comment]
@pernielsentikaer 

- editor can now be set by preference
- update package.json

Please review again!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**